backend/src/main/java/de/neuefische/backend/model/weatherapi/WeatherApiWeather.java

```diff
@@ -0,0 +1,16 @@
+package de.neuefische.backend.model.weatherapi;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class WeatherApiWeather {
```

Comment:
too much weather imho 

```suggestion
public class ApiWeather {
```